test/verify/check-login

```diff
@@ -104,15 +104,15 @@ account    required     pam_succeed_if.so user ingroup %s""" % m.get_admin_group
         b.wait_text('#current-username', 'admin')
 
         if m.image not in ['fedora-coreos']: # logs in via ssh, not cockpit-session
-            self.assertRegex(m.execute("who"), r"(^|\n)admin *web.*172.27.0.2")
+            self.assertRegex(m.execute("who"), r"(^|\n)admin *web.*::")
```

Comment:
Could we keep this IPv4 compatible? For example, with `\d+[.:]\d+` . I'm not sure if the machines in Fedora/RHEL gating speak IPv6.

Reply:
As per the screenshot in #14874, Fedora ends up being `::1`.  RHEL is another question.

Reply:
ah, that wouldn't work with `::1`. Maybe `(\d+\.\d+|::)`?

Reply:
Took your suggestion, though.  Will make this more robust against all kinds of possible environments.  Thanks!